mlrun/serving/server.py

```diff
@@ -339,9 +338,9 @@ def v2_serving_init(context, namespace=None):
         **kwargs,
     )
     context.logger.info("Initializing graph steps")
-    serving_handler = server.init_object(namespace or get_caller_globals())
+    server.init_object(namespace or get_caller_globals())
     # set the handler hook to point to our handler
-    setattr(context, "mlrun_handler", serving_handler)
+    setattr(context, "mlrun_handler", v2_serving_handler)
```

Comment:
Is it always sync?
If yes, do you still plan to change it to async in some cases?

Reply:
It will return a coroutine in case that the underlying implementation is async.